runtime/doc/ui.txt

```diff
@@ -170,6 +171,14 @@ option by itself will never cause any additional grids to be created.
 Highlight attribute groups are predefined. UIs should maintain a table to map
 numerical highlight `id`:s to the actual attributes.
 
+TODO: rewrite:
+By default, Nvim will only draw windows on one single grid, just like a
+terminal emulator. But when multigrid mode is activated Nvim might use
+multiple grids. Currently this is only used for floting windows |ui-float|,
```

Comment:
`floting` typo?

Reply:
Thanks. Most of this paragraph is obsolete...